queue/queue.py

```diff
@@ -1,15 +1,18 @@
 class Queue:
-  def __init__(self):
-    self.size = 0
-    # what data structure should we
-    # use to store queue elements?
-    self.storage = 
+    def __init__(self):
+        self.size = 0
+        self.storage = []
 
-  def enqueue(self, item):
-    pass
-  
-  def dequeue(self):
-    pass
+    def enqueue(self, item):
+        self.storage.append(item)
```

Comment:
Good implemtation, but it is also possible to import and utilize Linked Lists for that. For example instead of `append` You could use `add_to_tail` pre-built LL method that is avaiable in the Linked List.